deny.toml

```diff
@@ -26,15 +26,35 @@ license-files = [
 [advisories]
 ignore = [
     # `net2` crate has been deprecated; use `socket2` instead
+    # Solution: No safe upgrade is available!
+    # net2 v0.2.34
     "RUSTSEC-2020-0016",
-    # failure: failure is officially deprecated/unmaintained
+
+    # Type confusion if __private_get_type_id__ is overriden
+    # Solution: No safe upgrade is available!
+    # failure v0.1.8
+    "RUSTSEC-2019-0036",
     "RUSTSEC-2020-0036",
+
+    # dirs is unmaintained, use dirs-next instead
+    # Solution: No safe upgrade is available!
+    # dirs v2.0.2
+    "RUSTSEC-2020-0053",
+
     # stdweb is unmaintained
-    # can be removed after next release of time
+    # Solution: No safe upgrade is available!
+    # stdweb v0.4.20
+    # can be removed after next release of `time` crate
     # https://github.com/time-rs/time/commit/8b2734351a5cd05a9bd3315f1af76c28b5852fec
     "RUSTSEC-2020-0056",
-    # ordered_flat - can be removed when heim next releases their beta branch
-    "RUSTSEC-2020-0082",
-    # lucet-runtime-internals - we'll need to upgrade here
-    "RUSTSEC-2020-0004",
+
+    # Some lock_api lock guard objects can cause data races
+    # Solution: Upgrade to >=0.4.2
```

Comment:
Perhaps we should add the reason why we can't "just" upgrade this right now?

Reply:
Yes, I'll add more notes and create relevant issues for advisories ID's